packages/calcite-components/src/components/tile-select/tile-select.tsx

```diff
@@ -30,6 +30,7 @@ import { TileSelectType } from "./interfaces";
 import { CSS } from "./resources";
 
 /**
+ * @deprecated  Use the `calcite-tile` component instead.
```

Comment:
nitpick:

It looks like there's two spaces between the `@deprecated` and `"Use"`.

Reply:
🦅👁️ Good catch!